butterfree/migrations/database_migration/database_migration.py

```diff
@@ -68,6 +64,23 @@ def _get_diff(
             db_schema: object that contains the table of a given db schema.
 
         """
+        if not db_schema:
+            create_columns = dict()
+            for fs_item in fs_schema:
+                create_columns.update(
+                    {
+                        fs_item.get("column_name"): (
+                            fs_item.get("type"),
+                            fs_item.get("primary_key"),
+                        )
+                    }
+                )
+            schema_diff = set(
+                Diff(str(col), kind=Diff.Kind.CREATE, value=value)
+                for col, value in create_columns.items()
+            )
+            return schema_diff
+
```

Comment:
It's kinda redundant to make a list of Diffs if every single feature column of the feature set schema will be there and the value is just the properties of each feature, also present in the schema. See my comment above.